core/ledger/kvledger/txmgmt/privacyenabledstate/test_exports.go

```diff
@@ -118,11 +120,11 @@ func (env *CouchDBCommonStorageTestEnv) setupCouch() string {
 
 // Init implements corresponding function from interface TestEnv
 func (env *CouchDBCommonStorageTestEnv) Init(t testing.TB) {
-	redoPath, err := ioutil.TempDir("", "pestate")
+	redoPath, err := ioutil.TempDir("", strings.ReplaceAll(fmt.Sprintf("test-env-%s-", t.Name()), "/", "-"))
 	if err != nil {
```

Comment:
not yours but could you do a follow-up to convert these to use `assert`? thx

Reply:
replaced by 	assert.NoError(t, err)